docs/ai/how-to/app-service-aoai-auth.md

```diff
@@ -49,7 +49,7 @@ Your application can be granted two types of identities:
 Run the `az webapp identity assign` command to create a system-assigned identity:
 
 ```azurecli
-az webapp identity assign --name <app-name> --resource-group <group-name>
+az webapp identity assign --name {appName} --resource-group {groupName}
```

Comment:
Where did you get the guidance to use curly braces? At least one style guide we use says to use the angle brackets:

<img width="461" alt="image" src="https://github.com/dotnet/docs/assets/24882762/02041ebb-5bb7-4c8c-a8de-3dd84de08d53">

"We discourage curly braces { } for use as syntactical placeholders. Readers can confuse curly brace placeholders with the same notation used in:

- Replaceable text
- Format strings
- String interpolation
- Text templates
- Similar programming constructs

Consider using curly braces instead of angle brackets when angle brackets are normally used in the syntax of a language, such as in XML or HTML."

Reply:
I hadn't seen that guidance yet, thanks for sharing. I just used a syntax similar to C# string interpolation, but I can see how that might cause confusion.